packages/core/contracts/umips/VotingUpgrader.sol

```diff
@@ -26,23 +26,31 @@ contract VotingUpgrader {
     // Addresses to upgrade.
     address public newVoting;
 
+    // Address to call setMigrated on the old voting contract.
+    address public setMigratedAddress;
+
     /**
      * @notice Removes an address from the whitelist.
      * @param _governor the Governor contract address.
      * @param _existingVoting the current/existing Voting contract address.
      * @param _newVoting the new Voting deployment address.
      * @param _finder the Finder contract address.
+     * @param _setMigratedAddress the address to set migrated. This address will be able to continue making calls to
+     *                            old voting contract (used to claim rewards on others' behalf). Note: this address
+     *                            can always be changed by the voters.
      */
     constructor(
         address _governor,
         address _existingVoting,
         address _newVoting,
-        address _finder
+        address _finder,
+        address _setMigratedAddress
```

Comment:
you could just use `msg.sender` for this, no? I guess it's more generalised in how you've done it now and in future we might want to set the migrated address to the new voting contract so not really any downside of doing it this way.

Reply:
Good point. We could. But yeah, I thought it might be better to keep it explicit and agnostic of the deployer.